proto/import_sstpb.proto

```diff
@@ -234,6 +234,7 @@ message WriteResponse {
 message RawWriteBatch {
     uint64 ttl = 1;
     repeated Pair pairs = 2;
+    uint64 ts = 3;
```

Comment:
All pairs have the same timestamp ?

Reply:
Where does `ts` come from ?
If `ts` is specified by request, the ingest data would be covered by earlier PUTs, and can not be CDC.
Another alternative is generated by TiKV.

Reply:
The `ts` should be generated by PD's TSO. `RawWriteBatch` would be sent to all the peers of a region concurrently. If `ts` is generated by TiKV, the batches would have different `ts` underlay.

Reply:
Then what will happen if underlaying `ts` are different ?

Reply:
If the timestamp is different, from the perspective of MVCC, a `ts` can read different versions of keys from different TiKV, especially during leader change.

Reply:
But RawKV gets do not provide a snapshot of a given `ts`. It always return the latest version.